Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,77 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ButtonContainer from './components/ButtonComponents/ButtonContainer';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayValue: '',
+      number1: '',
+      operator: '',
+      number2: ''
+    };
+    this.calculate = {
+      '+': function(x, y) {return x + y},
+      '-': function(x, y) {return x - y},
+      'x': function(x, y) {return x * y},
+      '÷': function(x, y) {return x / y}
+    }
+  }
+
+  changeDisplayValue(value) {
+    console.log('hey');
+    if ((this.state.number1 === '') && (value !== ('+' || '-' || 'x' || '÷'))) {
+      this.setState({
+        number1: value
+      })
+      console.log(value);
+    }
+    if (value === '') {
+      this.setState({
+        displayValue: ''
+      })
+    }
+    if (value === ('+' || '-' || 'x' || '÷')) {
```

Comment:
Remember that this (`'+' || '-' || 'x' || '÷'`) isn't going to work like you want it to in JS! It'll short circuit and default to the first truthy value, in this case `'+'`, so your comparison actually ends up being `value === '+'`. The same applies elsewhere in your code where you try to use this pattern.